Lib/ast.py

```diff
@@ -1349,7 +1349,11 @@ def visit_Subscript(self, node):
         self.set_precedence(_Precedence.ATOM, node.value)
         self.traverse(node.value)
         with self.delimit("[", "]"):
-            if isinstance(node.slice, Tuple) and node.slice.elts:
+            if (
+                isinstance(node.slice, Tuple)
+                and node.slice.elts
+                and not any(isinstance(elt, Starred) for elt in node.slice.elts)
+            ):
```

Comment:
This conditional is getting a bit bigger than it should, could you refactor it into a function (defined here) so we can use the name in the conditional?